### PR TITLE
Change docker client to the official one

### PR DESCRIPTION
**Pull request by noseglid**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ script:
   - go test -v -race -coverprofile=coverage.txt -covermode=atomic -tags=integration
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+  - go get github.com/mattn/goveralls
+  - goveralls -coverprofile=coverage.txt
```
Comment (domszyn) [inline .travis.yml:17]: 🙏

Comment (johan-lejdung) [reply .travis.yml:17]: What's wrong with codecov 😄

File: initiator.go [hunk 2]
```diff
@@ -19,66 +26,68 @@ type ContainerConfig struct {
 	Tmpfs         map[string]string
 }
 
-var obsoleteAfter float64 = 10 * 60 // in seconds
+var ObsoleteAfter = 10 * time.Minute
```
Comment (johan-lejdung) [inline initiator.go:29]: Nice change to `time.Duration` 👍

File: mysql/mysql_probe.go [hunk 3]
```diff
@@ -37,10 +37,10 @@ func (i MysqlProbe) DoProbe(instance *dockerinitiator.Instance) error {
 	}()
 
 	db, err := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s)/%s", "root", i.Password, instance.GetHost(), i.DbName))
-	defer db.Close()
 	if err != nil {
 		return err
 	}
+	defer db.Close()
```
Comment (johan-lejdung) [inline mysql/mysql_probe.go:43]: I've had this error before 😅 
Can be a bit annoying and hard to find.

I did a git blame and saw that it was me who put it there... 🙃

**Review by johan-lejdung — APPROVED** [review]
Nice! Looks a lot cleaner 👌

I hope you all are doing well!

File: .travis.yml (hunk 1 above)
Comment (noseglid) [reply .travis.yml:17]: Pricing model. We've decided to switch to coveralls because of that :)

File: initiator.go (hunk 2 above)
Comment (noseglid) [reply initiator.go:29]: Yep, I'm pretty sure I built this before understanding the duration model in Go :)

File: mysql/mysql_probe.go (hunk 3 above)
Comment (noseglid) [reply mysql/mysql_probe.go:43]: Hehe, simple mistake to make! Luckily the Open hasn't errored that often yet!

**Review by domszyn — APPROVED** [review]

**Pull request merged**